os/drivers/audio/alc5658.c

```diff
@@ -264,7 +270,7 @@ static void alc5658_writereg(FAR struct alc5658_dev_s *priv, uint16_t regaddr, u
 
 	lower_hword = (regaddr >> 8) | ((regaddr & 0xFF) << 8);
 	high_hword = (regval >> 8) | ((regval & 0xFF) << 8);
-	full_word = (int32_t)(((int32_t)high_hword << 16) | ((int32_t)lower_hword));
+	full_word = (int32_t)(((int32_t) high_hword << 16) | ((int32_t) lower_hword));
```

Comment:
no previous is correct rule for our platform

Reply:
Reverted.